app/src/main/java/ch/epfl/sweng/SDP/firebase/Connection.java

```diff
@@ -0,0 +1,17 @@
+package ch.epfl.sweng.SDP.firebase;
+
+import android.content.Context;
+import android.net.ConnectivityManager;
+import android.net.NetworkInfo;
+
```

Comment:
Maybe you could have a better name for this class or add a documentation of what the class is doing :)

Reply:
Sure, do you have an idea for a name?

Reply:
The class name should describe what the class does. So maybe ConnectivityTester?